src/EditorFeatures/CSharpTest/UseLocalFunction/UseLocalFunctionTests.cs

```diff
@@ -3100,7 +3100,39 @@ public void Caller()
         U local(U x) => x;
         Callee(local);
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseLocalFunction)]
+        [WorkItem(23149, "https://github.com/dotnet/roslyn/issues/29793")]
+        public async Task TestExtraTrailingClosingBracket()
+        {
+            await TestInRegularAndScriptAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        Func<Task> [||]getTask = () =>
+        {
+            return Task.CompletedTask;
```

Comment:
Note: you're missing a using for System.Threading.Tasks, so this code is actually full of errors. It might be fine though if you verified that the test originally failed and now it passes.

Reply:
yes, i verified it, but i think i need to simplify it, thanks